web/src/components/cards/PackCard/PackCard.tsx

```diff
@@ -0,0 +1,213 @@
+      setIsUpdating(false);
+      trackError({
+        event: TrackErrorEnum.FailedToUpdatePack,
+        src: SrcEnum.Packs,
+      });
+      pushSnackbar({
+        variant: 'error',
+        title: `Failed to update Pack`,
+        description: extractErrorMessage(error),
+      });
+    },
+  });
+
+  const onPatch = (values: UpdateVersionFormValues) => {
+    setIsUpdating(true);
+    return updatePack({
+      variables: {
+        input: {
+          id: pack.id,
+          packVersion: values.packVersion,
+        },
+      },
+    });
+  };
+
+  const onStatusUpdate = (values: UpdateStatusFormValues) => {
+    setIsUpdating(true);
+    return updatePack({
+      variables: {
+        input: {
+          id: pack.id,
+          enabled: values.enabled,
+        },
+      },
+    });
+  };
+  return (
+    // Replaced GenericItemCard with simple card in order to exclude overflow property
+    <Card as="section" variant="dark" position="relative">
+      {isUpdating && (
+        <Flex
+          position="absolute"
+          direction="column"
+          spacing={2}
+          backgroundColor="navyblue-700"
+          height="100%"
+          zIndex={2}
+          alignItems="center"
+          opacity={0.9}
+          justify="center"
+          width={1}
+        >
+          <Text textAlign="center" opacity={1}>
+            The pack is updating, please wait.
+          </Text>
+          <BulletedLoading />
+        </Flex>
+      )}
+      <Flex position="relative" height="100%" p={4}>
+        <GenericItemCard.Body>
+          <GenericItemCard.Header>
+            <GenericItemCard.Heading>
+              <Link as={RRLink} aria-label="Link to Pack" to={urls.packs.details(pack.id)}>
+                {pack.displayName || pack.id}
+              </Link>
+            </GenericItemCard.Heading>
+            <Flex spacing={2} fontSize="small" alignItems="center">
+              {pack.updateAvailable && (
+                <Box
+                  as="span"
+                  backgroundColor="red-500"
+                  borderRadius="small"
+                  px={2}
+                  py={1}
+                  fontWeight="bold"
+                >
+                  UPDATE AVAILABLE
+                </Box>
+              )}
+              <Text as="span">Version</Text>
+              <Text as="span" color="gray-400">
+                {pack.packVersion.name}
+              </Text>
+            </Flex>
+          </GenericItemCard.Header>
+          <Flex spacing={2}>
+            <Box
+              as="span"
+              backgroundColor="navyblue-700"
+              borderRadius="small"
+              p={1}
+              fontSize="small"
+              color="cyan-500"
+              mr={1}
+            >
+              {pack.detectionTypes.RULE} Rules
+            </Box>
+            <Box
+              backgroundColor="navyblue-700"
+              borderRadius="small"
+              p={1}
+              fontSize="small"
```

Comment:
All labels should have different typography

```suggestion
              fontWeight="bold"
              fontSize="x-small"
```